src/WinRT.Runtime/Projections/ICustomPropertyProvider.net5.cs

```diff
@@ -23,6 +25,105 @@ interface ICustomProperty
         bool CanWrite { get; }
         string Name { get; }
         global::System.Type Type { get; }
+    }
+
+    /// <summary>
+    /// An interface complementing <see cref="BindableCustomPropertyAttribute"/> providing the implementation to expose the specified properties.
+    /// </summary>
+#if EMBED
+    internal
```

Comment:
Nit: given this is all new code, perhaps we can enable nullability annotations around it? Both the interface and the property type. Ideally, the attribute as well (since eg. the properties can be `null` when the default constructor is used).

Reply:
I took an attempt at properly supporting nullable here.  If we were to annotate it, I would need to annotate the Funcs too which would mean things like the value possibly being null and the index possibly being null.  And then using that, I enabled nullable in the generated code and tried to properly annotate that, but we run into scenarios where we will need to then also detect if the user code types are nullable or not and based on that generate the appropriate code.  Given this interface and class are both to support just the generated code, is just an implementation detail, and not meant to be called directly by consumers, I am going to skip annotating these.